docs/recipes/watch-mode.md

```diff
@@ -59,7 +59,9 @@ In AVA there's a distinction between *source files* and *test files*. As you can
 
 By default AVA watches for changes to the test files, `package.json`, and any other `.js` files. It'll ignore files in [certain directories](https://github.com/novemberborn/ignore-by-default/blob/master/index.js) as provided by the [`ignore-by-default`] package.
 
-You can configure patterns for the source files using the [`--source` CLI flag] or in the `ava` section of your `package.json` file. Note that if you specify a negative pattern the directories from [`ignore-by-default`] will no longer be ignored, so you may want to repeat these in your config.
+You can configure patterns for the source files using the [`--source` CLI flag] or in the `ava` section of your `package.json` file.
```

Comment:
Maybe we should put `in the `ava` section of your `package.json` file` and mention it's the recommended way. Having the config in package.json will be important for future editor plugins and other integrations as they won't be able to read the config from the npm run script.